deploy/helm/flightctl/templates/flightctl-alertmanager-proxy-serviceaccount.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.alertmanagerProxy.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    flightctl.service: flightctl-alertmanager-proxy
+  name: flightctl-alertmanager-proxy
+  namespace: {{ default .Release.Namespace .Values.global.internalNamespace }}
```

alertmanager-proxy will always be deployed to `.Release.Namespace` [see](https://github.com/flightctl/flightctl/pull/1452), SA needs to be in the same ns

```suggestion
  namespace: {{ .Release.Namespace }}
```

---

Fixed